File: c/PRAK405-2510817210027-Ahmad_Tibrizi.c
#include <stdio.h>

int main() {
    int looping, kelipatan;
    int temp_hasil = 0;
    int hasil = 0;

    printf("Banyak perulangan: ");
    scanf("%d", &looping);
    printf("Banyak kelipatan: ");
    scanf("%d", &kelipatan);

    for (int i = 1; i <= looping; i++) {
        temp_hasil += i * kelipatan;
        
        for (int j = 1; j <= i; j++) {
            printf("(%d * %d)", j, kelipatan);
            if (j < i) {
                printf(" + ");
            }
        }
        printf(" = %d\n", temp_hasil);
        hasil += temp_hasil;
    }

    printf("%d\n", hasil);

    return 0;
}

File: python/PRAK405-2510817210027-Ahmad_Tibrizi.py
looping = int(input("Banyak perulangan: "))
kelipatan = int(input("Banyak kelipatan: "))
temp_hasil = 0
hasil = 0
rumus = ''

for i in range(1, looping+1):
    temp_hasil += i * kelipatan
    hasil += temp_hasil

    if i == looping:
        rumus += f'({i} * {kelipatan}) = {temp_hasil}'
        print(rumus)
        print(hasil)
    else:
        rumus += f'({i} * {kelipatan})'
        print(f'{rumus} = {temp_hasil}')
        rumus += ' + '
